samples/TodoList/src/ts/services/ServiceRegistrar.ts

```diff
@@ -0,0 +1,23 @@
+/**
+* ServiceRegistrar.tsx
```

Comment:
You moved this code here, but also have it in the base appbootstrapper

Reply:
Not sure what you mean. The ServiceRegistrar is registering all services. The AppBootstrapper is starting a few of them (well, actually all of them in the current sample).